Tests/Unit/Model/Transaction_Test.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+class Transaction_Test extends OxidEsales\TestingLibrary\UnitTestCase
+{
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $oOrder = oxNew(\OxidEsales\Eshop\Application\Model\Order::class);
+        $oOrder->save();
+
+        $oDb = oxDb::getDb();
```

Comment:
I guess this actually uses the database to execute the statement below. I believe we should find a way to mock this. Unit tests should not have such dependencies / side-effects.

Reply:
you are right. I'll move it to integration tests because I acually test the database.